tfhe/src/core_crypto/gpu/mod.rs

```diff
@@ -260,6 +259,56 @@ pub unsafe fn convert_lwe_keyswitch_key_async<T: UnsignedInteger>(
     dest.copy_from_cpu_multi_gpu_async(src, streams);
 }
 
+/// Applies packing keyswitch on a vector of LWE ciphertexts
+///
+/// # Safety
+///
+/// [CudaStreams::synchronize] __must__ be called as soon as synchronization is
+/// required
+#[allow(clippy::too_many_arguments)]
+pub unsafe fn packing_keyswitch_list_async<T: UnsignedInteger>(
```

Comment:
nit: if it matches the rest of the GPU naming this is fine, but it does not match the CPU API naming

Reply:
It follows since I don't think we are using a strong convention for this bind related methods. e.g. keyswitch is jsut `keyswitch_async`. Probably we should improve that. I will open an issue.